InternetHospitalWebUI/src/app/Components/Authorization/sign-in/sign-in.component.ts

```diff
@@ -42,7 +41,12 @@ export class SignInComponent implements OnInit {
         .pipe(first())
         .subscribe(
             data => {
-                this.router.navigate([this.returnUrl]);
+                if(this.authenticationService.hasAdminRole()){
+                    this.router.navigate(['/supersecurityadminpanel']);
```

Comment:
It would be better to create some consts for ways, because if we change name of route in route-module, it would crash.